frontend/sass/evictionfree/_bulma-overrides.scss

```diff
@@ -0,0 +1,40 @@
+// Fallbacks taken from:
+// http://melchoyce.github.io/fontstacks/examples/open-sans.html
+
+// FONTS:
+$family-sans-serif: $jf-body-family;
+
+// COLORS:
+$justfix-green: pink;
```

Comment:
This just sets the main green accent color to pink for now... I wanted to set it to something other than green to differentiate it from the norent site at least.  I think this is the setting that Tahnee is going to decide a more appropriate color for.